### ISSUE TITLE
Hi! Your results look really promising! Any chance the code will be provided?

### ISSUE THREAD

**Issue by muxgt** [Zheng222/DMFN#1]

**Comment by Zheng222** [comment]
@muxgt Of course, testing and training codes will be released successively.  Thanks for your interest.

**Comment by muxgt** [comment]
@Zheng222 Cool, thanks! Any approximate dates already?

**Comment by zimenglan-sysu-512** [comment]
glad to see the release of code

**Comment by heartInsert** [comment]
Your  result  looks  very  nice , I  can't  waitting  anymore ,  I  want  to  see  your code .

Xiong Di.

**Comment by tvshow5727** [comment]
I am looking forward to seeing the release of code,When do you update the code?
thanks a lot....
